src/app/client/src/app/modules/public/module/player/components/public-content-player/public-content-player.component.ts

```diff
@@ -86,6 +86,13 @@ export class PublicContentPlayerComponent implements OnInit, OnDestroy, AfterVie
    * @memberof ContentPlayerComponent
    */
   ngOnInit() {
+    /** It will handle device back-button click to rotate landscape to portrait */
+    window.addEventListener('orientationchange', () => {
```

Comment:
@HostListener('window:orientationchange', ['$event']) can be used instead

Reply:
@anoophm   Updated the PR